diff --git a/node_modules/acorn/dist/acorn.js b/node_modules/acorn/dist/acorn.js
index e2b3317..35ae260 100644
--- a/node_modules/acorn/dist/acorn.js
+++ b/node_modules/acorn/dist/acorn.js
@@ -2285,7 +2285,7 @@
     if (!this.eat(types.parenR)) {
       var errorPos = this.start;
       if (this.eat(types.comma) && this.eat(types.parenR)) {
-        this.raiseRecoverable(errorPos, "Trailing comma is not allowed in import()");
+        this.raiseRecoverable(errorPos, "Trailing comma is not allowed in import call");
       } else {
         this.unexpected(errorPos);
       }
@@ -2405,7 +2405,7 @@
     var startPos = this.start, startLoc = this.startLoc, isImport = this.type === types._import;
     node.callee = this.parseSubscripts(this.parseExprAtom(), startPos, startLoc, true);
     if (isImport && node.callee.type === "ImportExpression") {
-      this.raise(startPos, "Cannot use new with import()");
+      this.raise(startPos, "Cannot use new with import call");
     }
     if (this.eat(types.parenL)) { node.arguments = this.parseExprList(types.parenR, this.options.ecmaVersion >= 8, false); }
     else { node.arguments = empty$1; }
@@ -4484,7 +4484,7 @@
     }
     if (next === 33 && code === 60 && !this.inModule && this.input.charCodeAt(this.pos + 2) === 45 &&
         this.input.charCodeAt(this.pos + 3) === 45) {
-      // `<!--`, an XML-style comment that should be interpreted as a line comment
+      // an XML-style comment that should be interpreted as a line comment
       this.skipLineComment(4);
       this.skipSpace();
       return this.nextToken()
