diff --git a/node_modules/acorn-node/lib/dynamic-import/index.js b/node_modules/acorn-node/lib/dynamic-import/index.js
index bb37198..aafa83a 100644
--- a/node_modules/acorn-node/lib/dynamic-import/index.js
+++ b/node_modules/acorn-node/lib/dynamic-import/index.js
@@ -28,7 +28,7 @@ function _inherits(subClass, superClass) { if (typeof superClass !== "function"
 
 var DynamicImportKey = exports.DynamicImportKey = 'Import';
 
-// NOTE: This allows `yield import()` to parse correctly.
+// NOTE: This allows `yield import` to parse correctly.
 _acorn.tokTypes._import.startsExpr = true;
 
 function parseDynamicImport() {
