diff --git a/node_modules/ethereumjs-abi/node_modules/keccak/index.js b/node_modules/ethereumjs-abi/node_modules/keccak/index.js
index 7446e03..bb62b01 100644
--- a/node_modules/ethereumjs-abi/node_modules/keccak/index.js
+++ b/node_modules/ethereumjs-abi/node_modules/keccak/index.js
@@ -1,10 +1 @@
-'use strict'
-try {
-  module.exports = require('./bindings')
-} catch (err) {
-  if (process.env.DEBUG) {
-    console.error('Keccak bindings are not compiled. Pure JS implementation will be used.')
-  }
-
-  module.exports = require('./js')
-}
+module.exports = require('./js')
