diff --git a/node_modules/secp256k1/index.js b/node_modules/secp256k1/index.js
index 8ccc453..5e06c07 100644
--- a/node_modules/secp256k1/index.js
+++ b/node_modules/secp256k1/index.js
@@ -1,10 +1 @@
-'use strict'
-try {
-  module.exports = require('./bindings')
-} catch (err) {
-  if (process.env.DEBUG) {
-    console.error('Secp256k1 bindings are not compiled. Pure JS implementation will be used.')
-  }
-
-  module.exports = require('./elliptic')
-}
+module.exports = require('./elliptic')
