diff --git a/node_modules/secp256k1/build/Makefile b/node_modules/secp256k1/build/Makefile
index 098bd11..9c005a2 100644
--- a/node_modules/secp256k1/build/Makefile
+++ b/node_modules/secp256k1/build/Makefile
@@ -308,8 +308,8 @@ ifeq ($(strip $(foreach prefix,$(NO_LOAD),\
 endif
 
 quiet_cmd_regen_makefile = ACTION Regenerating $@
-cmd_regen_makefile = cd $(srcdir); /home/xyz/.nvm/versions/node/v10.18.1/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py -fmake --ignore-environment "--toplevel-dir=." -I/tmp/tmp-18033tpvyb05A1aC0/node_modules/secp256k1/build/config.gypi -I/home/xyz/.nvm/versions/node/v10.18.1/lib/node_modules/npm/node_modules/node-gyp/addon.gypi -I/home/xyz/.cache/node-gyp/10.18.1/include/node/common.gypi "--depth=." "-Goutput_dir=." "--generator-output=build" "-Dlibrary=shared_library" "-Dvisibility=default" "-Dnode_root_dir=/home/xyz/.cache/node-gyp/10.18.1" "-Dnode_gyp_dir=/home/xyz/.nvm/versions/node/v10.18.1/lib/node_modules/npm/node_modules/node-gyp" "-Dnode_lib_file=/home/xyz/.cache/node-gyp/10.18.1/<(target_arch)/node.lib" "-Dmodule_root_dir=/tmp/tmp-18033tpvyb05A1aC0/node_modules/secp256k1" "-Dnode_engine=v8" binding.gyp
-Makefile: $(srcdir)/../../../../home/xyz/.nvm/versions/node/v10.18.1/lib/node_modules/npm/node_modules/node-gyp/addon.gypi $(srcdir)/../../../../home/xyz/.cache/node-gyp/10.18.1/include/node/common.gypi $(srcdir)/build/config.gypi $(srcdir)/binding.gyp
+cmd_regen_makefile = cd $(srcdir); /home/xyz/.nvm/versions/node/v10.18.1/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py -fmake --ignore-environment "--toplevel-dir=." -I/home/xyz/Development/eth-faucet/node_modules/secp256k1/build/config.gypi -I/home/xyz/.nvm/versions/node/v10.18.1/lib/node_modules/npm/node_modules/node-gyp/addon.gypi -I/home/xyz/.cache/node-gyp/10.18.1/include/node/common.gypi "--depth=." "-Goutput_dir=." "--generator-output=build" "-Dlibrary=shared_library" "-Dvisibility=default" "-Dnode_root_dir=/home/xyz/.cache/node-gyp/10.18.1" "-Dnode_gyp_dir=/home/xyz/.nvm/versions/node/v10.18.1/lib/node_modules/npm/node_modules/node-gyp" "-Dnode_lib_file=/home/xyz/.cache/node-gyp/10.18.1/<(target_arch)/node.lib" "-Dmodule_root_dir=/home/xyz/Development/eth-faucet/node_modules/secp256k1" "-Dnode_engine=v8" binding.gyp
+Makefile: $(srcdir)/../../../../.nvm/versions/node/v10.18.1/lib/node_modules/npm/node_modules/node-gyp/addon.gypi $(srcdir)/build/config.gypi $(srcdir)/binding.gyp $(srcdir)/../../../../.cache/node-gyp/10.18.1/include/node/common.gypi
 	$(call do_cmd,regen_makefile)
 
 # "all" is a concatenation of the "all" targets from all the included
diff --git a/node_modules/secp256k1/build/Release/obj.target/secp256k1.node b/node_modules/secp256k1/build/Release/obj.target/secp256k1.node
index 7b0d82e..7356006 100755
Binary files a/node_modules/secp256k1/build/Release/obj.target/secp256k1.node and b/node_modules/secp256k1/build/Release/obj.target/secp256k1.node differ
diff --git a/node_modules/secp256k1/build/Release/obj.target/secp256k1/src/ecdh.o b/node_modules/secp256k1/build/Release/obj.target/secp256k1/src/ecdh.o
index 011e2c5..89945a4 100644
Binary files a/node_modules/secp256k1/build/Release/obj.target/secp256k1/src/ecdh.o and b/node_modules/secp256k1/build/Release/obj.target/secp256k1/src/ecdh.o differ
diff --git a/node_modules/secp256k1/build/Release/obj.target/secp256k1/src/ecdsa.o b/node_modules/secp256k1/build/Release/obj.target/secp256k1/src/ecdsa.o
index 7134e28..43bf255 100644
Binary files a/node_modules/secp256k1/build/Release/obj.target/secp256k1/src/ecdsa.o and b/node_modules/secp256k1/build/Release/obj.target/secp256k1/src/ecdsa.o differ
diff --git a/node_modules/secp256k1/build/Release/obj.target/secp256k1/src/privatekey.o b/node_modules/secp256k1/build/Release/obj.target/secp256k1/src/privatekey.o
index 2d1f503..affc763 100644
Binary files a/node_modules/secp256k1/build/Release/obj.target/secp256k1/src/privatekey.o and b/node_modules/secp256k1/build/Release/obj.target/secp256k1/src/privatekey.o differ
diff --git a/node_modules/secp256k1/build/Release/obj.target/secp256k1/src/publickey.o b/node_modules/secp256k1/build/Release/obj.target/secp256k1/src/publickey.o
index 4fe5444..c44f13e 100644
Binary files a/node_modules/secp256k1/build/Release/obj.target/secp256k1/src/publickey.o and b/node_modules/secp256k1/build/Release/obj.target/secp256k1/src/publickey.o differ
diff --git a/node_modules/secp256k1/build/Release/obj.target/secp256k1/src/signature.o b/node_modules/secp256k1/build/Release/obj.target/secp256k1/src/signature.o
index a30d88b..af5b902 100644
Binary files a/node_modules/secp256k1/build/Release/obj.target/secp256k1/src/signature.o and b/node_modules/secp256k1/build/Release/obj.target/secp256k1/src/signature.o differ
diff --git a/node_modules/secp256k1/build/Release/secp256k1.node b/node_modules/secp256k1/build/Release/secp256k1.node
index 7b0d82e..7356006 100755
Binary files a/node_modules/secp256k1/build/Release/secp256k1.node and b/node_modules/secp256k1/build/Release/secp256k1.node differ
diff --git a/node_modules/secp256k1/index.js b/node_modules/secp256k1/index.js
index 8ccc453..5e06c07 100644
--- a/node_modules/secp256k1/index.js
+++ b/node_modules/secp256k1/index.js
@@ -1,10 +1 @@
-'use strict'
-try {
-  module.exports = require('./bindings')
-} catch (err) {
-  if (process.env.DEBUG) {
-    console.error('Secp256k1 bindings are not compiled. Pure JS implementation will be used.')
-  }
-
-  module.exports = require('./elliptic')
-}
+module.exports = require('./elliptic')
